config/manifest.yaml

```diff
@@ -3,7 +3,6 @@ dist:
   name: otelcol-custom
   module: github.com/liatrio/liatrio-otel-collector
   description: Basic OTel Collector distribution for Developers
-  otelcol_version: 0.113.0
```

Comment:
This config value was removed https://github.com/open-telemetry/opentelemetry-collector/issues/11405
Now this defaults to the version of the builder that compiles the collector.